js/apps/admin-ui/src/realm-settings/localization/EffectiveMessageBundles.tsx

```diff
@@ -62,15 +67,21 @@ export const EffectiveMessageBundles = ({
     Partial<EffectiveMessageBundlesSearchForm>
   >({});
   const themes = serverInfo.themes;
-  const themeKeys = themes
-    ? Object.keys(themes).sort((a, b) => a.localeCompare(b))
+  const themeTypes = themes
+    ? localeSort(
+        Object.keys(themes).filter((key) => key !== "common"),
+        (key) => key,
+      )
+    : [];
+  const themeNames = themes
+    ? localeSort(
+        Object.values(themes)
+          .flatMap((theme) => theme.map((item) => item.name))
+          .filter((value, index, self) => self.indexOf(value) === index),
+        (name) => name,
+      )
     : [];
```

Comment:
Likewise here.
```suggestion
  const themeNames = useMemo(() => {
    if (!themes) {
      return [];
    }
  
    return localeSort(
      Object.values(themes)
        .flatMap((theme) => theme.map((item) => item.name))
        .filter((value, index, self) => self.indexOf(value) === index),
      (name) => name,
    );
  }, [themes]);
```

Reply:
Thanks @jonkoops you're 100% right here!